src/main/java/lotto/domain/Statistics.java

```diff
@@ -0,0 +1,37 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import lotto.domain.vo.RankCounts;
+
+public class Statistics {
+
+    private final LottoRanks lottoRanks;
+    private final Map<LottoRank, Integer> rankCounts;
+
+    public Statistics(final LottoRanks lottoRanks) {
+        this.lottoRanks = lottoRanks;
+        this.rankCounts = new LinkedHashMap<>();
+
+        initRankCount();
+    }
+
+    private void initRankCount() {
+        Arrays.stream(LottoRank.values())
+                .forEach(rank -> rankCounts.put(rank, 0));
+    }
+
+    public RankCounts getRankCounts() {
+        lottoRanks.get()
+                .forEach(rank -> rankCounts.put(rank, rankCounts.get(rank) + 1));
+
+        return new RankCounts(rankCounts);
+    }
+
+    public double getProfitRate() {
+        final List<LottoRank> lottoRanks = this.lottoRanks.get();
+        return lottoRanks.stream().mapToDouble(LottoRank::getAmount).sum() / (lottoRanks.size() * 1000);
+    }
```

Comment:
```suggestion
    public double getProfitRate() {
      double a = lottoRanks.get().stream().mapToDouble(LottoRank::getAmount).sum();
      double b = (lottoRanks.get().size() * 1000);

       return a / b;
   }
```

변수로 받아 주고 `lottoRanks.get()`에서 `get()`을 없애줄 수 있지 않을까요?